src/main/java/moriyashiine/aylyth/common/item/AylythItems.java

```diff
@@ -1,26 +1,35 @@
 package moriyashiine.aylyth.common.item;
 
 import com.terraformersmc.terraform.boat.impl.item.TerraformBoatItem;
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import moriyashiine.aylyth.common.Aylyth;
 import moriyashiine.aylyth.common.block.AylythBlocks;
 import moriyashiine.aylyth.common.entity.AylythEntityTypes;
 import moriyashiine.aylyth.common.entity.AylythStatusEffects;
-import moriyashiine.aylyth.common.item.type.*;
-import moriyashiine.aylyth.common.item.component.ThornFlechetteEffect;
-import moriyashiine.aylyth.common.other.*;
-import moriyashiine.aylyth.common.other.util.AylythUtil;
+import moriyashiine.aylyth.common.item.types.*;
+import moriyashiine.aylyth.common.item.components.ThornFlechetteEffect;
+import moriyashiine.aylyth.common.util.AylythUtil;
+import moriyashiine.aylyth.common.world.AylythSoundEvents;
 import net.fabricmc.fabric.api.item.v1.FabricItemSettings;
+import net.fabricmc.fabric.api.itemgroup.v1.FabricItemGroup;
 import net.minecraft.block.Block;
 import net.minecraft.entity.EntityType;
 import net.minecraft.entity.effect.StatusEffectInstance;
 import net.minecraft.entity.mob.MobEntity;
 import net.minecraft.item.*;
 import net.minecraft.registry.*;
+import net.minecraft.text.Text;
 import net.minecraft.util.Rarity;
 import net.minecraft.util.math.Direction;
 
+import java.util.List;
+
 public interface AylythItems {
 
-	Item DEBUG_WAND = register("debug_wand", new DebugWandItem(settings()));
+    List<Item> TEMPT_MAIN_ITEM_GROUP_ITEMS = new ObjectArrayList<>();
```

Comment:
Mutable collection that should not be publicly accessible

Reply:
Before PR this field was also public mutable collection, but now I can't make it private, the interfaces don't support it. I will leave a comment in the source code

Reply:
Hm, so it was. Never should have been. The fix would be to make it a regular class again, but it can be ignored for now